File: programacion actividades/ejerciciosU2/src/EJ4.java
import java.lang.annotation.Repeatable;

import java.util.Scanner;


public class EJ4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Introduce un numero y te dare la raiz cuadrada");

        int num = sc.nextInt();

        System.out.println("La raiz es : " + Math.sqrt(num));


    }
}

File: KOTLIN U7/untitled/src/Ej4.kt
import java.util.Scanner

fun main() {
    val Scanner = Scanner(System.`in`)
    var contador = 0
    print("Introduce un primo : ")



    val num = Scanner.nextInt()



    for (i in 1..num){
        if (num % i == 0){
            contador++
        }else{

        }
    }
    if (contador > 2){
        print("El numero $num no es primo")
    }else {
        print("El numero $num es primo ")
    }

}